R/utils.R

```diff
@@ -162,3 +162,17 @@ getFileExtension <- function(filePath) {
   )
   return(extension)
 }
+
+
+#' @title loadSimulationWithUpdatedPaths
+#' @param simulationSet simulation set containing path to simulation file and pathIDs for quantities to be loaded into simulation object
+#' @return simulation object with pathIDs updated from simulationSet
+loadSimulationWithUpdatedPaths <- function(simulationSet) {
+  sim <- ospsuite::loadSimulation(filePath = simulationSet$simulationFile)
+  sim$outputSelections$clear()
+  quantities <- getAllQuantitiesMatching(paths = simulationSet$pathID, container = sim)

```

Comment:
use the `addOutputs` method from ospsuite-R
```
#' paths <- c("Organism|VenousBlood|Plasma|Caffeine", "Organism|ArterialBlood|**|Caffeine")
#' addOutputs(paths, sim)
```

Reply:
done